src/lib/components/compositions/dialog/dialog.story.svelte

```diff
@@ -1,4 +1,5 @@
 <script lang="ts">
+  /* eslint @typescript-eslint/no-empty-function: "off" */
```

Comment:
I'm conditionally turning off this ESLint rule within some `story` or `stories` files as some components require a "on click" or "on change" function that I don't need in our stories. I'd like the ability to just pass `() => {}` as the function.